spec/requests/skateparks_spec.rb

```diff
@@ -30,6 +43,17 @@
     expect(json_body['lon']).to eq(other.lon)
   end
 
+  it 'returns users that have favorited skatepark' do
+    park = create(:skatepark)
+    user = create(:user)
+    park.users << user
+
+    get "/skateparks/#{park.id}"
+
+    expect(json_body['users'][0]['name']).to eq(user.name)
+    expect(json_body['users'][0]['email']).to eq(user.email)
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.